components/Graphs/ForksChart.tsx

```diff
@@ -27,8 +27,11 @@ type ForksChartProps = {
 
 export default function ForksChart({ stats, total, syncId, range = 30 }: ForksChartProps) {
   const [category, setCategory] = useState<"daily" | "cumulative">("daily");
-  const dailyData = getDailyForksHistogramToDays({ stats, range });
-  const cumulativeData = getCumulativeForksHistogramToDays({ stats, total, range });
+  const dailyData = useMemo(() => getDailyForksHistogramToDays({ stats, range }), [stats, range]);
```

Comment:
Aside from fixing the flippening, this doesn't perform these expensive calculations unless the related data changes.